.ci.yaml

```diff
@@ -445,17 +445,16 @@ targets:
         ]
 
   - name: Mac impeller-cmake-example
-    bringup: true
+    #bringup: true
```

Comment:
I will uncomment this before landing. It's uncommented to force it to run in the presubmit checks.